lib/confirmation.rb

```diff
@@ -0,0 +1,28 @@
+require 'twilio-ruby'
+require_relative 'basket'
+
+class ConfirmText
+
+  def send_text
+    initiate_msg(conf_irm)
+  end
+
+  private
+
+  def initiate_msg(msg)
+    account_sid = ENV['TWILIO_ACCOUNT_SID']
+    auth_token = ENV['TWILIO_AUTH_TOKEN']
+  # set up a client to talk to the twilio REST API
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+  # send an sms
+    @client.messages.create(
+    from: ENV['TWILIO_PHONE'],
+    to: ENV['TWILIO_DESTINATION_PHONE'],
+    body: msg)
+  end
+
+  def conf_irm
```

Comment:
If I understand correctly, your intention is to separate the message content from the action of sending it.  That's great! So you could extent this class to do something like `initiate_message(cancelled)` inside a method called `send_cancellation_notice` perhaps and include the content for that message in a method called `cancelled`.

Can you go one step further than this?

Is there a way you could encapsulate only the message content, somewhere else, and then use that here?  You could then do something like this `intiate_message(MESSAGES[:confirm])`.

What do you think?  What would that encapsulation look like?  Would it be another class?  Could it be something else?